### DISCUSSION TITLE
0.122.0 expected release date?

### DISCUSSION THREAD

**Discussion by pisarenko91** [zed-industries/zed#7405] [Feature Requests]
Hey team, when can we expect 0.122.0 in preview? Thanks a lot.

**Comment by maxbrunsfeld** [comment]
We ship new minor versions every Wednesday. This Wednesday, we'll ship v0.122.0 preview, and v0.121.3 stable.

**Reply by pisarenko91** [reply]
Thank you very much for your quick reply.
